### DISCUSSION TITLE
452 users in docker variant

### DISCUSSION THREAD

**Discussion by VasilyBorisovich** [photoprism/photoprism#2773] [Q&A] [answered]
Why do you need 4 hundreds of users? )
![452_user](https://user-images.githubusercontent.com/33516500/195104836-c8df38b4-d7ba-4bed-b772-489e2be32ff1.jpg)

**Comment by lastzero** [comment, marked as answer]
These accounts could have been created dynamically, but then users find ways to bypass the entrypoint script and run the image on appliances that work with exactly one of these user IDs, otherwise everything is broken - leading to time-consuming support requests. Of course, if we had nothing else to do, there might be an equally reliable but better solution. Feel like working on it? Note that there are slight differences between these user accounts.
